mlflow/sklearn/__init__.py

```diff
@@ -654,6 +655,17 @@ def fetch_logged_data(run_id):
     :param silent: If ``True``, suppress all event logs and warnings from MLflow during scikit-learn
                    autologging. If ``False``, show all events and warnings during scikit-learn
                    autologging.
+    :param max_tuning_runs: The maximum number of child Mlflow runs to be created for
+                            hyperparameter search estimators. If one only cares about
+                            creating an mlflow child run for the best k results(based on
+                            `rank_test_score` values) from the search, then set max_tuning_runs
```

Comment:
Can we remove (`based on rank_test_score values`) and move this discussion about how the best values are selected to the bottom of the docstring along with the information about custom scorers? These two are closely related.